rules/bigquery_rules.yaml

```diff
@@ -24,7 +24,7 @@ rules:
     resource:
       - type: organization
         resource_ids:
-          - {ORGANIZATION_ID}
+          - 826592752744
```

Comment:
Ooops, shouldn't be using specific id values here.  Please change back to the placeholder?